tracelog/tracelog_test.go

```diff
@@ -217,7 +217,7 @@ func TestLogCopyFrom(t *testing.T) {
 		return config
 	}
 
-	pgxtest.RunWithQueryExecModes(context.Background(), t, ctr, nil, func(ctx context.Context, t testing.TB, conn *pgx.Conn) {
+	pgxtest.RunWithQueryExecModes(context.Background(), t, ctr, pgxtest.KnownOIDQueryExecModes, func(ctx context.Context, t testing.TB, conn *pgx.Conn) {
```

Comment:
@jackc this is the tests that was making the CI fail. It only fails on cockroachDB, and when retrieving the types from the typeMap instead of a prepared statement.

When using a prepare statement it fails because there's a type mismatch between the input which is a `string` and the OID from the prepared statement. 

When using the types from the typeMap, the OID matches the input, but the input doesn't match the table definition.  but no error is returned.

Reply:
Maybe this has something to do with my comment above.

Reply:
The CI hadn't finished and I was seeing the previous error 🤦